.flake8

```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 160
```

Comment:
https://github.com/googleapis/google-cloud-python/blob/master/container/.flake8 should be used as a template.